src/Components/Publishing/Byline/DateSource.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { Fragment } from "react"
+import styled from "styled-components"
+import colors from "../../../Assets/Colors"
+import { pMedia } from "../../Helpers"
+import { getDate } from "../Constants"
+import { Fonts } from "../Fonts"
+import { NewsBylineProps } from "./NewsByline"
+
+interface Props {
+  editSource?: any
+}
+
+export const DateSource: React.SFC<NewsBylineProps & Props> = props => {
+  const { article, editSource } = props
+  const { news_source, published_at } = article
+
+  const getNewsSource = source => {
+    if (!source || !source.url) return null
+    return (
+      <Fragment>
+        {", via"}&nbsp;
+        {editSource ? editSource : <a href={source.url}>{source.title}</a>}
+      </Fragment>
```

Comment:
Usually `editSouce` is a function, so would probably need to be called here

Reply:
Hmm it's a component so I think it can be returned as is. To verify I looked at some other places we're doing the same thing and we don't call it. Am I missing something?

Reply:
That might be an older version that uses children, here is a recent example:
https://github.com/artsy/reaction/blob/master/src/Components/Publishing/Sections/Image.tsx#L46

And this is the positron side:
https://github.com/artsy/positron/blob/master/client/apps/edit/components/content/sections/images/components/edit_image.jsx#L62

Reply:
Ah the reason that one is a func is because we want to do a conditional in Positron -- that makes sense. But this recent example renders the component: https://github.com/artsy/reaction/blob/master/src/Components/Publishing/Series/SeriesTitle.tsx#L24

Is func a preference for you? It seems like there are varied usages and that we haven't established a standard.

Reply:
Oh yes i see -- in that case this looks good!